core/Object.php

```diff
@@ -285,10 +285,16 @@ protected static function _parents() {
 	/**
 	 * Exit immediately. Primarily used for overrides during testing.
 	 *
+	 * @deprecated This method will be removed in a future version. Please use
```

Comment:
There's one blank line too much. Also I think you can remove all the double \\ for a single \